### ISSUE TITLE
Add POI layer

### ISSUE THREAD

**Issue by cgiovando** [hotosm/osm-analytics#30]
Also mentioned in #14 - to add another layer in addition to roads and buildings density. Assuming we can pull this from the same OSM QA Tile source.

As requested by @TylerRadford - this would be really useful to support some ongoing HOT projects.

@tyrasd - let us know if this is something quick that you can add for the beta launch this week.

Thanks!

**Comment by tyrasd** [comment]
Which POIs are you thinking about, or do you mean POIs in general like all `amenity=*` objects?

**Comment by cgiovando** [comment]
Yes, I think general POIs would be a good to start, then later we could make sub-categories. Thanks!

**Comment by tyrasd** [comment]
OK, here's a first go at this: http://osm-analytics.org/#/show/bbox:-117.56401,32.59299,-116.47774,32.99219/pois/recency

I've not made the POIs layer selectable from the layer-chooser at the top right because of the following reasons:
* We don't have the historic renderings for the compare view for it yet
* It includes only osm nodes (Point geometries) wíth an `amenity=*` tag. All other kinds of POIs and POIs on polygons are not included
* It's currently not integrated in the daily cron update

Here's the [job](https://github.com/hotosm/osm-analytics-cruncher/#runsh-path-to-osmqatilesmbtiles-job-binningfactor) definition [file](https://gist.github.com/tyrasd/755000de1ba8c1f994793c526ebdfdbb) I've used for this test.

**Comment by jjiglesiasg** [comment]
Places could be added as a Feature? This is important to measure the development of the map in different regions,,,
